tests/checkout/nasty.c

```diff
@@ -10,6 +10,24 @@ static const char *repo_name = "nasty";
 static git_repository *repo;
 static git_checkout_options checkout_opts;
 
+#ifdef GIT_WIN32
+static bool is8dot3Disabled(void)
+{
+	char src[MAX_PATH];
+	wchar_t dest[MAX_PATH], shortPath[MAX_PATH];
+	FILE *fp;
+	strcpy_s(src, MAX_PATH, clar_sandbox_path());
+	strcat_s(src, MAX_PATH, "/longer_than_8dot3");
```

Comment:
If we end up with a path which is longer than `MAX_PATH`, this is going to crash the test suite.